src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -64,6 +66,8 @@ def __init__(
         self.ring_size = ring_size
 
         self.config = config
+        self.fp_encoder = None
```

Comment:
MAYBE: Remove this

Reply:
I know it is not the scope of this PR - but could we also remove the `AUTOGRAD_IS_ON` from line 40 (I think that attribute is now in the MPC Level)